hikari/impl/shard.py

```diff
@@ -553,6 +553,12 @@ async def get_user_id(self) -> snowflakes.Snowflake:
             raise RuntimeError("user_id was not known, this is probably a bug")
         return self._user_id
 
+    def user_id(self) -> snowflakes.Snowflake:
+        if not self._user_id:
+            raise errors.ComponentStateConflictError("User ID not known yet")
+
+        return self._user_id
```

Comment:
With https://github.com/hikari-py/hikari/pull/1204 this will no longer be necessary, as you can just do `get_user_id`.

Is it worth merging that one before and rebasing onto it?